src/test/java/uk/gov/hmcts/reform/cpo/validators/IdsValidatorTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.cpo.validators;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import javax.validation.ConstraintValidatorContext;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class IdsValidatorTest {
```

Comment:
In theory this work duplicated by incorrectly named IdsTest.  At some point they probably should be consolidated and named properly.